labelbox/schema/project.py

```diff
@@ -352,6 +352,47 @@ def create_prediction(self, label, data_row, prediction_model=None):
         res = self.client.execute(query_str, params)
         return Prediction(self.client, res["createPrediction"])
 
+    def upload_annotations(
+            self,
+            name: str,
+            url: str = None,
+            path: str = None,
+            annotations: Iterable[dict] = None
+    ) -> None:
+
+        exclusionary_args = iter((url, path, annotations))
+        # any will stop iteration when it finds a single truthy value
+        # not any will continue with where the iteration left off and
+        # check the rest are falsy
+        assert (any(exclusionary_args)
+                and not any(exclusionary_args)), (
+            'Only one of url, path, or annotations can be specified.'
+            )
```

Comment:
1. I prefer raising `ValueError` over assertions for invalid args. 
2. `any(exclusionary_args) and not any(exclusionary_args)` <- it'll be hard to make it true:D